src/VisualStudio/Setup/VisualStudioSetup.csproj

```diff
@@ -140,6 +140,7 @@
   </ItemGroup>
   <ItemGroup>
     <InternalsVisibleTo Include="Microsoft.VisualStudio.LanguageServices.Telemetry" />
+    <InternalsVisibleTo Include="Roslyn.VisualStudio.Setup.Next" />
```

Comment:
If you're using this to consume the redirection attributes, that's not quite safe: the DLL that you're then consuming through will break. Take a look at the previous bug we had around loading Roslyn.VisualStudio.Setup that I and @jmarolf fixed, you'll see we linked the source no matter what to play it safe.